### PR TITLE
MOBILE-4268: Apply design system to Error Accordion

### PR DESCRIPTION
**Pull request by NoelDeMartin**

### CODE REVIEW

File: src/core/components/error-info/error-info.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class CoreErrorInfoComponent implements OnInit, OnChanges {
     }
 
     @Input() errorDetails!: string;
-    @Input() errorCode?: string;
+    @Input() errorCode!: string;
```
Comment (crazyserver) [inline src/core/components/error-info/error-info.ts]: Is this ! permitted or it throws a compilation warning,

File: src/core/components/error-accordion/error-accordion.ts [hunk 2]
```diff
@@ -23,11 +23,11 @@ import { CoreForms } from '@singletons/form';
  * it can be rendered using the static render() method to get the raw HTML.
  */
 @Component({
-    selector: 'core-error-info',
-    templateUrl: 'core-error-info.html',
-    styleUrls: ['error-info.scss'],
+    selector: 'core-error-accordion',
```
Comment (crazyserver) [inline src/core/components/error-accordion/error-accordion.ts:31]: We should probably deprecate the previous component, nit just remove It. Probably admiting both d'electors van be enough.

Comment (crazyserver) [reply src/core/components/error-accordion/error-accordion.ts:31]: However se should revise with which components de are invoking with selectors and which creating them in TS code

File: src/core/components/error-accordion/error-accordion.ts [hunk 3]
```diff
@@ -32,41 +37,91 @@ export class CoreErrorAccordionComponent implements OnInit, OnChanges {
...
+                    </div>
+                    <ion-icon name="chevron-down" />
+                </button>
             </div>
         `;
     }
 
-    @Input() errorDetails!: string;
+    /**
+     * Hydrage component.
```
Comment (crazyserver) [inline src/core/components/error-accordion/error-accordion.ts:87]: Typo

**Review by crazyserver — CHANGES_REQUESTED** [review]

File: src/core/components/error-accordion/error-accordion.ts (hunk 2 above)
Comment (NoelDeMartin) [reply src/core/components/error-accordion/error-accordion.ts]: I refactored the code to use a service instead, given that this isn't really an Angular Component. It was a component before to use it in storybook, but given that we've removed it now, it's no longer necessary.

**Pull request merged**